app/build.gradle

```diff
@@ -41,6 +41,12 @@ android {
         }
     }
 
+    configurations.all {
+        resolutionStrategy {
+            force 'org.hamcrest:hamcrest-core:2.2'
```

Comment:
enforce the use of 2.2 version of a hamcrest-core due to version conflicts between dependencies